book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```

Comment:
- [ ] I think you should use Getters and Setters for `@rentals` because of the many-to-many relationships between the rental and the book.